test/snapshots/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, util, snapshots }:
+
+with stdenv.lib;
+with util;
+
+let
+  env = snapshots."lts-12.21".ghcWithHoogle
+    (ps: with ps; [ conduit conduit-extra resourcet ]);
+
+in
+  stdenv.mkDerivation {
+    name = "shell-for-test";
+
+    buildCommand = ''
+      ########################################################################
+      # test snapshot ghcWithHoogle
+
+      printf "checking that the ghcWithPackages env has the package...\n" >& 2
+      ${env}/bin/ghc-pkg list | grep conduit
+
+      touch $out
+    '';
+
+    meta.platforms = platforms.all;
+    passthru = {
+      # Used for testing externally with nix-shell (../tests.sh).
```

Comment:
`nix-shell (../tests.sh)`? Shouldn't that be `nix-shell test/snapshots/default.nix`? I'm a bit confused by that comment.

Reply:
I fixed it -- the comment doesn't apply for this test because it's not one of the impure `tests.sh` test cases.